Tests/SwiftGenKitTests/StoryboardsMacOSTests.swift

```diff
@@ -64,4 +64,18 @@ class StoryboardsMacOSTests: XCTestCase {
 
     XCTDiffContexts(result, expected)
   }
+
+  func testAllStoryboardsWithCustomName() {
+    let parser = StoryboardParser()
+    do {
+      try parser.parseDirectory(at: Fixtures.directory(sub: .storyboardsMacOS))
+    } catch {
+      print("Error: \(error.localizedDescription)")
+    }
+
+    let result = parser.stencilContext(sceneEnumName: "XCTStoryboardsScene", segueEnumName: "XCTStoryboardsSegue")
```

Comment:
Shouldn't we use this PR to migrate `stencilContext(sceneEnumName:, segueEnumName: )` to `stencilContext(params: […])` instead, since the `--sceneEnumName` and `--segueEnumName` flags of `swiftgen` are also deprecated in favour of `--param`?